pyproject.toml

```diff
@@ -46,6 +46,7 @@ includes = "src"
 
 [tool.pytest.ini_options]
 asyncio_mode = "auto"
+testpaths = ["tests"]
```

Comment:
I noticed that Pytest was trying to discover tests in folders such as src/ or sandbox/, so let's limit its search scope 🙂